src/_internal/icon/src/Icon.tsx

```diff
@@ -19,11 +20,28 @@ export default defineComponent({
       type: String,
       required: true
     },
+    size: {
```

Comment:
Do not add size prop to BaseIcon since it will make inner svg inherit the color of the wrapper dom. Set size by inline style (using `font-size`) in Rate instead.

For example <n-base-icon style="font-size: 14px;" /> will make the inner svg of size 14px.